block/block.md

```diff
@@ -1 +1,105 @@
 # Block
+
+## Abstract
+
+The block manager is a key component of full nodes and is responsible for block production or block syncing depending on the node type. Block syncing in this context includes retrieving the published blocks from the network (p2p network or DA network), validating them to raise fraud proofs upon validation failure, updating the state, and storing the validated blocks. A full node invokes multiple block manager functionalities in parallel, such as:
```

Comment:
We should name the spec Block Manager if that is what we are describing.

Reply:
whats the right name? i think block is still valid as it will include various services like block management, block sync, header sync, etc?

Reply:
It's just that you never talk about the block, only ever the services.  Plus you say that management includes syncing. Currently header sync is separate from block sync, but that feels like an implementation detail and it is reasonable for the block manager to be responsible for all parts of the block, and header is part of the block. The block being a struct/type, that the block manager handles.

Reply:
may be we can defer this for later?

Reply:
I think Sevey just means to change block.md to block-manager.md